src/Files/MultilingualResources/Files.nl-NL.xlf

```diff
@@ -3073,231 +3073,231 @@ We gebruiken App Center om app-gebruik bij te houden, bugs te vinden en crashes
         </trans-unit>
         <trans-unit id="ConfictingItemsDialogOptionApplyToAll.Text" translate="yes" xml:space="preserve">
           <source>Apply to all</source>
-          <target state="new">Apply to all</target>
+          <target state="new">Toepassen op alles</target>
```

Comment:
Can you update the state to `translated`? https://files.community/docs/contributing/translating-files